src/host/os_isdir.c

```diff
@@ -9,15 +9,15 @@
 #include "premake.h"
 
 #ifdef _WIN32
-#include <Windows.h>
+#include <windows.h>
```

Comment:
Was this causing build issues? If not, I'd consider it out of scope of this PR.

Reply:
Compiler error:

```
src/host/os_isdir.c:12:10: fatal error: Windows.h: No such file or directory
   12 | #include <Windows.h>
      |          ^~~~~~~~~~~
```